README.adoc

```diff
@@ -22,7 +22,7 @@ Mutual TLS means that connecting clients need to have a client certificate befor
 |Client certificates will expire, and hence need to recreate these for all clients periodically, which introduces a lot of overhead and potential for errors
 
 |
-|Have to maintain a CA for signing client certs
+|Have to maintain a CA for signing client certs. (Have you thought about how you will revoke access should this need arise?)
```

Comment:
For internal communication I have a reasonably nice solution in LEV using long-lived, self-signed certs.
There's not too much benefit over standard TLS plus basic auth, except that the auth happens outside of HTTP. (It's possibly you would want to use basic auth for something else.)

Any I would like to get people's thoughts on it...